notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>A description of the output in terms of a dictionaries with keys and values explain may help here. We understand that from lines 15 to 17 in the code cell, but having a markdown cell above with the structure clearly described may help.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>